README.md

```diff
@@ -9,31 +9,23 @@ npm install ethfs-uploader
 ## Usage
 ### Support EIP-3770 Address
 ```
-mainnet 
-    w3q:<domain/address>
-
-galileo
-    w3q-g:<domain/address>
-
 ethereum
     eth:<domain/address>
 
-rinkeby
-    rin:<domain/address>
+... 
+
+galileo
+    w3q-g:<domain/address>       
 ```
 ##### Example
 ```
-mainnet
-    w3q:home.w3q
-
-galileo
-    w3q-g:0x1825...2388
-
 ethereum
     eth:ens.eth
```

Comment:
As Qi said, the example should list all the networks we supported, check out the list [here](https://docs.web3url.io/advanced-topics/web3url-gateways).  So we may need to deploy the factory contract on those networks first, then embed them in ethfs-uploader